builtin/providers/aws/resource_aws_route53_health_check_test.go

```diff
@@ -0,0 +1,164 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+
+	"github.com/awslabs/aws-sdk-go/service/route53"
+)
+
+func TestAccRoute53HealthCheck(t *testing.T) {
```

Comment:
We'll need to rename this to something like `TestAccRoute53HealthCheck_basic`, so that it can be ran independently from `TestAccRoute53HealthCheck_IpConfig`